v2/expectation.go

```diff
@@ -93,8 +93,8 @@ func (t *NthTest) Desc() string {
 
 // Do implements Expectation
 func (t *NthTest) Do(ctx context.Context, resp Response) (err error) {
-	root, err := lzjson.Decode(resp.Body())
-	if err != nil {
+	root := lzjson.Decode(resp.Body())
```

Comment:
This change looks like the right thing to do when I look at the [Decode function definition](https://github.com/go-restit/lzjson/blob/39aa0b4f12586747f4b4233e1c9101f56b919057/node.go#L81-L87)